### PR TITLE
Re-export types from fflate

### PR DESCRIPTION
**Pull request by Methuselah96**
### Why

We should just depend on `fflate` and re-export the types from there instead of them duplicating them here.

### What

Re-export types from `fflate`.

### Checklist

-   [x] Checked the target branch (current goes `master`, next goes `dev`)
-   [x] Added myself to contributors table
-   [x] Ready to be merged

### CODE REVIEW

**Comment by Methuselah96** [comment]
Waiting for https://github.com/microsoft/DefinitelyTyped-tools/pull/627 to get released.

File: types/three/package.json [hunk 1]
```diff
@@ -10,5 +10,8 @@
         "./addons/*": "./examples/jsm/*",
         "./src/*": "./src/*",
         "./nodes": "./examples/jsm/nodes/Nodes.js"
+    },
+    "dependencies": {
+        "fflate": "~0.6.9"
```
Comment (joshuaellis) [inline types/three/package.json]: Does webxr not need to be here then? 🤔

Comment (Methuselah96) [reply types/three/package.json]: IIRC, any `@types/*` packages get added into the `dependencies` at publish-time by DefinitelyTyped, and therefore don't need to be specified here.

Comment (joshuaellis) [reply types/three/package.json]: Cool! thanks for explaining 😄

**Review by joshuaellis — APPROVED** [review]

**Pull request merged**